src/server/storage/memory.js

```diff
@@ -100,6 +101,69 @@ function Memory(mainLogger, gmeConfig) {
             return deferred.promise.nodeify(callback);
         };
 
+        this.loadPaths = function (rootHash, paths, excludes, callback) {

```

Comment:
This pretty complex piece of code is copied here and in Mongo. Couldn't it be implemented in the storage instead?